index.html

```diff
@@ -0,0 +1,136 @@
+          <h1>JS Documentation</h1>
+        </header>
+        <div class="nav-wrapper">
+          <ul>
+            <li>
+              <a class="nav-link" href="#Introduction">Introduction </a>
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#what_you_should_already_know"
+                >what you should already know</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#JavaScript_and_Java"
+                >JavaScript and Java</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#Hello_world">Hello world</a>
+              <hr />
+            </li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
+          <h2>Introduction</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="what_you_should_already_know" class="main-section">
+          <h2>what you should already know</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="JavaScript_and_Java" class="main-section">
+          <h2>JavaScript and Java</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+        </section>
+        <section id="Hello_world" class="main-section">
+          <h2>Hello world</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
```

Comment:
This paragraph spans more than 100 characters. So, please limit its width like the ones above.